backend/migrator/migration/3.10/0004##add_project_labels.sql

```diff
@@ -0,0 +1,5 @@
+-- Add labels column to project table for storing key-value labels
```

Comment:
Instead of introducing a new column, let's shelf everything to project proto instead. Some SQL fields exist for historical reasons.